judges/quality-of-service/quality-of-service.rb

```diff
@@ -32,15 +32,25 @@
   total = 0
   success = 0
   duration = 0
+  mttrs = []
+  mttr = {}
   Fbe.unmask_repos.each do |repo|
     Fbe.octo.repository_workflow_runs(repo, created: ">#{f.since.utc.iso8601[0..9]}")[:workflow_runs].each do |json|
```

Comment:
@tank-bohr probably, you're right. But I think it would be better to fix it through another pull request. Because this is beyond the scope of the current task.